### PR TITLE
#168135490 Implement full navbar

### PR DESCRIPTION
**Pull request by Nerocodes**
#### Pivotal tracker story
[#168135490 ](https://www.pivotaltracker.com/story/show/168135490 )

#### What does this PR do?
- Implement full navbar

#### Summary of Task
- Implement full navbar

#### How can this be tested?
- Git clone
- Run ``` npm install ```
- Run ``` npm run dev ```

##### Authenticated
image

##### Unauthenticated
image

### CODE REVIEW

File: webpack.config.development.js [hunk 1]
```diff
@@ -26,17 +26,9 @@ module.exports = {
           },
           {
             loader: 'css-loader',
-            options: {
```
Comment (Nkemjiks) [inline webpack.config.development.js:29]: Why did you take this out

Comment (Nerocodes) [reply webpack.config.development.js:29]: It stopped us from importing our scss directly and we had to put it in a variable like this:
``` import Stlye from './style.scss' ``` which meant our classes on components had to be ``` Style.header ```. I discussed with the team before making changes

**Comment by Nkemjiks** [comment]
I will need bella to use this commit in his PR. its kind of hard to navigate stand alone

**Comment by Nerocodes** [comment]
> I will need bella to use this commit in his PR. its kind of hard to navigate stand alone

@Nkemjiks the sign in and sign up are going to be modals so they currently don't navigate anywhere. I thought it'll be better suited for those working on those pages

**Comment by Nkemjiks** [comment]
> > I will need bella to use this commit in his PR. its kind of hard to navigate stand alone
> 
> @Nkemjiks the sign in and sign up are going to be modals so they currently don't navigate anywhere. I thought it'll be better suited for those working on those pages

True but the will need a place to start from and that is the landing page. Bella needs it for his page to be complete

**Comment by Nkemjiks** [comment]
Can you close this PR @Nerocodes ?

**Pull request closed without merging**